src/Controls/src/Core/HandlerImpl/SearchBar.Impl.cs

```diff
@@ -0,0 +1,6 @@
+namespace Microsoft.Maui.Controls
+{
+	public partial class SearchBar : ISearchBar
```

Comment:
why are you adding this?

Reply:
Because the SearchBar PR did not add this or register it with the handlers. Seems I missed that when reviewing the PR. Without this it actually throes so I am not even sure how it ran locally...